graph/ui-icons/src/lib/technology-icon.tsx

```diff
@@ -1,33 +1,106 @@
 import { Framework, frameworkIcons } from './framework-icons';
+import { MonochromaticTechnologyIconsMap } from './technology-icons-map-monochromatic';
 
 export interface TechnologyIconProps {
   technology?: string;
   showTooltip?: boolean;
+  monochromatic?: boolean;
 }
 
 export function TechnologyIcon({
   technology,
   showTooltip,
+  monochromatic,
+}: TechnologyIconProps) {
+  if (!technology) {
+    return null;
+  }
+  if (monochromatic) {
+    return (
+      <TechnologyIconMonochromatic
+        technology={technology}
+        showTooltip={showTooltip}
+      />
+    );
+  }
+  return (
+    <TechnologyIconColor technology={technology} showTooltip={showTooltip} />
+  );
+}
+
+function UnknownTechnologyIcon({
+  technology,
+  showTooltip,
+}: TechnologyIconProps) {
+  if (!technology) {
+    return null;
+  }
+  return (
+    <div
+      className="flex h-4 w-4 items-center justify-center rounded bg-slate-800 text-sm text-slate-50 dark:bg-slate-50 dark:text-slate-800"
```

Comment:
While you're here, is it easy to make the icon in the project header (next to the Project Name) `h-6 w-6`? Right now it's really small next to the larger text.

![image](https://github.com/nrwl/nx/assets/8104246/cbe4f690-05f0-45e2-a49c-c7ef705169eb)

Reply:
changed 
![Screenshot 2024-05-03 at 12 29 18 AM](https://github.com/nrwl/nx/assets/16211801/52955d0d-60fa-437d-9ede-ea7f7369a5d0)
![Screenshot 2024-05-03 at 12 29 13 AM](https://github.com/nrwl/nx/assets/16211801/baca0b6c-0e83-4a93-82d4-1dd8f531e359)